.github/workflows/tests.yaml

```diff
@@ -41,9 +41,9 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.pyver }}
-      - run: pip install 'pip >=23' build
+      - run: sudo apt install python3-build
```

Comment:
Is `sudo` really needed? I'm wondering whether this container is already run as root.

Reply:
I tested without and the error message said that I wasn't root, so it's required sadly.

Reply:
the exact error message:
```
Run apt install python3-build

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

E: Could not open lock file /var/lib/dpkg/lock-frontend - open (13: Permission denied)
E: Unable to acquire the dpkg frontend lock (/var/lib/dpkg/lock-frontend), are you root?
Error: Process completed with exit code 100.
```